lib/smart_proxy_salt/cli.rb

```diff
@@ -64,13 +64,11 @@ def autosign_list
         def append_value_to_file(filepath, value)
```

Comment:
Another way to write this method:
```ruby
def append_value_to_file(filepath, value)
  File.open(filepath, File::CREAT|File::RDWR) do |file|
    unless file.any? { |line| line.chomp == value }
      file.puts(value)
    end
  end
  logger.info "Added an entry to '#{filepath}' successfully."
  true
rescue SystemCallError => e
  logger.info "Attempted to add an entry to '#{filepath}', but an exception occurred: #{e}"
  false
end
```

It has the benefit that it can stop early if the line has been found and uses fewer helper variables.

Reply:
Implemented as suggested :+1: